src/pcb/pcb_trace_hint.ts

```diff
@@ -11,6 +9,7 @@ export const pcb_trace_hint = z
     pcb_port_id: z.string(),
     pcb_component_id: z.string(),
     route: z.array(route_hint_point),
+    trace_width: distance,
```

Comment:
every point in a route can have an optional `trace_width` as well! can we modify the `route_hint_point` as well to add this optional property?

```suggestion
    route: z.array(route_hint_point),
    trace_width: distance.optional(),
```

Reply:
Okay!